apps/mobile/eas.json

```diff
@@ -20,9 +20,11 @@
     "maestro": {
       "extends": "base",
       "environment": "production",
-      "withoutCredentials": true,
```

Comment:
I needed to remove `"withoutCredentials": true,` to get around this problem in the Expo CI

![Screenshot 2025-06-25 at 12 04 46](https://github.com/user-attachments/assets/8c4525b2-a4e7-404a-9cdc-f024cf7067e3)